data/funds.json

```diff
@@ -78,7 +78,8 @@
         "dividendLeakage": 0,
         "kiid": "https://www.meesman.nl/media/1vhin3ml/ebi-miawta-2022.pdf",
         "factsheet": "https://www.meesman.nl/media/zpvkkdiw/factsheet-meesman-indexfonds-aandelen-wereldwijd-totaal-a-31-03-2022.pdf",
-        "index": "Meesman Aandelen Wereldwijd Totaal"
+        "index": "Meesman Aandelen Wereldwijd Totaal",
+        "esgExclusions": 6.3
```

Comment:
The ESG exclusions of Meesman (as well as the NT funds) are a little bit complicated at the moment.

The Meesman Indexfonds Aandelen Wereldwijd Totaal invests in three Northern Trust funds (World, EM & Small caps), which each follow a custom ESG index by MSCI. So neither the Meesman fund nor the NT funds do any exclusions themselves, instead the exclusions are done by the index.

The website doesn't reflect this really well at the moment, but for market cap calculation the exclusions are done when importing market cap data from index fact sheets. More specifically here: 

https://github.com/nicwortel/indexfondsenvergelijken.nl/blob/bf287741489512a427342666cf00e7355bfdf620/src/Import/MeesmanIndexFactory.php#L27-L31

Adding the 6.3% (which I think comes from the jaarverslag?) here would display that percentage on the website but would at the same time make the calculated market capitalization inaccurate because the ESG exclusions would be subtracted twice.

I need to find some time to rewrite this part of the website so I can better handle this situation, but for now I think it's better to leave the ESG exclusion out of this file.